message.go

```diff
@@ -324,6 +324,7 @@ func NewFixtureMessage(lang Lang, x Fixture, requestedAt int) *Message {
 			Lang:        lang,
 			ReceivedAt:  uniqTimestamp(),
 			RequestedAt: requestedAt,
+			Timestamp:   int(x.GeneratedAt.UnixMilli()),
```

Comment:
is it int64 or int32 ? be careful it has impact on read value

Reply:
int64, but it is defined as int on some parts of the library so I need that conversion.
Didn't see any issue so far.